compiler-rt/lib/builtins/cpu_model.c

```diff
@@ -1363,6 +1367,8 @@ static void __init_cpu_features_constructor(unsigned long hwcap,
     setCPUFeature(FEAT_SME_I64);
   if (hwcap2 & HWCAP2_SME_F64F64)
     setCPUFeature(FEAT_SME_F64);
+  if (hwcap2 & HWCAP2_SME_FA64)
+    setCPUFeature(FEAT_SME_FA64);
```

Comment:
We don't need FEAT_SME_FA64 in CPUFeatures enums if we are not supporting it in FMV.

Reply:
I think it is better to remove any code that become unused.

Reply:
Removed both this and the definition.

Reply:
FEAT_SME_FA64 still left in CPUFeatures in `cpu_model.c` and in `AArch64TargetParser.h` Remove it please.